source/data/search_serial_in_large_CRL.py

```diff
@@ -147,7 +147,7 @@ def main():
     print()
     print()
 
-    crl_filename = "www.sk.ee-esteid2011.crl"
+    crl_filename = "source/data/www.sk.ee-esteid2011.crl"
```

Comment:
Better revert this change that the script finds its data.

Reply:
Ah yes, I tried not to include the path changes but I missed them, I'll fix this soon.